CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Change Log
+# [0.64.0 (November 3rd, 2020)](https://github.com/eclipse/eclipse.jdt.ls/milestone/82?closed=1)
```

Comment:
Add https://github.com/eclipse/eclipse.jdt.ls/issues/1202 to changelog

Reply:
Done. Can someone add #1202 to milestone 82. I don't have permissions to do so.

Reply:
already added.